docs/sources/tasks/migrate/from-static.md

```diff
@@ -169,6 +172,23 @@ logs:
                       - level
       clients:
         - url: https://USER_ID:API_KEY@logs-prod3.grafana.net/loki/api/v1/push
+
+traces:
+  configs:
+    - name: tempo
+      receivers:
+        otlp:
+          protocols:
+            grpc:
+            http:
+      batch:
+        send_batch_size: 10000
+        timeout: 20s
+      remote_write:
+        - endpoint: tempo-us-central1.grafana.net:443
+          basic_auth:
+            username: USERNAME
+            password: PASSWORD
```

Comment:
```suggestion
            username: "<USERNAME>"
            password: "<PASSWORD>"
```

I'm guessing these need double quotes too?

Reply:
Honestly, I'm not sure. I'll defer to your judgement on this. @hedss Do you know?

Reply:
If what I found in the Static mode doc is correct, I don't think we need the double quotes. 
```suggestion
            username: <USERNAME>
            password: <PASSWORD>
```